components/organisms/footer.html

```diff
@@ -0,0 +1,97 @@
+<div class=footer>
+
+  <footer class="main-footer">
+
+    <div class="footer-button">
+      <h4>Example</h4>
+      <button type="button">Click here!</button>
+    </div>
+
+    <div class="social">
+      <h4>Connect with us!</h4>
+      <ul class="social-icons">
+        <li><a href="http://www.facebook.com"><img src='../../img/icon_facebook.png' /></a></li>
+        <li><a href="http://www.twitter.com"><img src='../../img/icon_instagram.png' /></a></li>
+        <li><a href="http://www.youtube.com"><img src='../../img/icon_twitter.png' /></a></li>
+      </ul>
+    </div>
+
+  </footer>
+
+</div>
+<style>
+
+  /* =================================
+  Base Layout Styles
+==================================== */
+  .footer {
+    padding-top: .75em;
+    background: rgba(232, 236, 241, 1);
+  }
+  .main-footer {
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+    grid-gap: 74px;
+    text-align: center;
+    padding: 2em 0;
+    background: rgba(54, 56, 57, .92);
+    text-align: left;
+    color: white;
+    font-family: arial;
+  }
+
+  .footer-button {
+    margin: 0 25px;
+  }
+
+  .social {
+    margin: 50px 25px;
+  }
+
+  button {
```

Comment:
As mentioned above, you may want to move these styles into a class like `.footer-button`. As written, this rule will style all buttons on a page, as opposed to only the footer buttons.

Reply:
You are absolutely correct @cocarson , I need to fix this one.